src/EA.Weee.Web/Views/OrganisationRegistration/RepresentingCompanyDetails.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿@using EA.Weee.Core.DataStandards
+@using EA.Weee.Web.Areas.Admin.Controllers
+@using MvcNavigationHelpers
+@using EA.Weee.Web.Controllers
+@model EA.Weee.Core.Organisations.RepresentingCompanyDetailsViewModel
+@{
+    ViewBag.Title = "Producer details";
+}
+@section main_content
+{
+    <header class="hgroup">
+        <h1 class="govuk-heading-l">Producer details</h1>
+    </header>
+
+    <p>Provide the address of the non-UK established Small EEE producer you are registering on behalf of.</p>
+    <p>If the producer has previously registered in the UK under the WEEE Regulations through being a member of a Producer Compliance Scheme, include their Producer Registration Number.</p>
+
+    @using (Html.BeginForm())
+    {
+        @Html.AntiForgeryToken()
+        @Html.Gds().ValidationSummary()
+        @Html.HiddenFor(m => m.OrganisationType)
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.CompanyName)">
+            @Html.Gds().LabelFor(m => m.CompanyName)
+            @Html.Gds().ValidationMessageFor(m => m.CompanyName)
+            @Html.Gds().TextBoxFor(m => m.CompanyName, new { @maxlength = CommonMaxFieldLengths.DefaultString })
+        </div>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.BusinessTradingName)">
+            @Html.Gds().LabelFor(m => m.BusinessTradingName)
+            @Html.Gds().TextBoxFor(m => m.BusinessTradingName, new { @maxlength = CommonMaxFieldLengths.DefaultString })
+        </div>
+
+        <div>
+            <h2 class="govuk-heading-s">Contact address</h2>
+        </div>
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.Address)">
+            @Html.EditorFor(m => m.Address)
+        </div>
+
+        <div class="govuk-form-group">
+            <button class="govuk-button" type="submit" data-module="govuk-button" data-prevent-double-click="true">Continue</button>
+        </div>
+        <p>
+            <a href="@(Url.UrlFor<OrganisationRegistrationController>(u => u.Type()))">Back</a>
+        </p>
```

Comment:
Need a back to top link. Use another view as example.

Reply:
updated